lib/nationbuilder/client.rb

```diff
@@ -44,6 +45,8 @@ def raw_call(path, method, body = {}, args = {})
       }
     }
 
+    request_args[:query][:fire_webhooks] = @fire_webhooks unless @fire_webhooks.nil?
```

Comment:
The only thing I'm suggesting is that we make it a parameter that applies per call instead of globally on the client object.